### PR TITLE
Read dev server from .env

### PR DESCRIPTION
**Pull request by schacon**
I think this addresses the dev problem I have with wanting to run the client against localhost, but it requires a `.env` file for everyone, it doesn't appear to be able to be left blank.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ then, make sure to install app dependencies:
 $ pnpm install
 ```
 
+### setup the env
```
Comment (ngalaiko) [inline README.md]: ideally, you shouldn't do that, see https://vitejs.dev/guide/env-and-mode.html#env-variables-and-modes

**Review by ngalaiko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (schacon) [reply README.md]: I'm not sure I follow. Is Vite a subset of Svelte? Or are you saying the env file management is the same? If there is a way you prefer to do this, I don't much care how it's done, I just need to be able to point my build to `http://localhost:3000` and have it fall back to test and prod builds point to production. How would you like to do this?

Comment (ngalaiko) [reply README.md]: svelte is built with vite, and it has integration to get env variables: https://kit.svelte.dev/docs/modules#$env-static-public - this is perhaps a better link

if your solution works, let's just merge it, this is not very important with only one variable, that's why i approved

Comment (ngalaiko) [reply README.md]: then, once you have `.env.development` and `.env.production`, make sure that `pnpm build` is ran with a correct `--mode` setting (should be correct by default), and your variables will be accessible as you want.

**Review by ngalaiko — APPROVED** [review]

**Pull request closed without merging**